src/alda/repl/commands/ReplLoad.java

```diff
@@ -83,8 +86,8 @@ private void loadFile(String args, AldaServer server, StringBuffer history,
 
       if (!error) {
         // Check if we can continue overwrite.
-        System.out.println("This action will overwrite the current score. Continue?");
-        if (Util.promptWithChoices(reader, Arrays.asList("yes", "no")) == "yes"){
+        if (Util.uncheckedPromptWithChoices(reader, OVERWRITE_WARNING,
+                                            "yes", "no") == "yes"){
```

Comment:
Usually this would be one of those places that comparing strings with `==` instead of `.equals()` causes bugs, but I tested this branch, and it actually works! I wonder if you can use `==` and `!=` to compare string values starting in Java 8?